src/font.js

```diff
@@ -235,6 +235,12 @@ Font.prototype.forEachGlyph = function(text, x, y, fontSize, options, callback)
             var kerningValue = this.getKerningValue(glyph, glyphs[i + 1]);
             x += kerningValue * fontScale;
         }
+
+        if (options.letterSpacing) {
+            x += options.letterSpacing * fontSize;
```

Comment:
Why multiply `letterSpacing` by `fontSize`?
If I try to get a path of a text with letterSpacing of 3px and a font size of 100px, this code will give me a spacing if 300, which is clearly not what I want. Am I missing something?
@lachmanski @fdb